digdag-ui/config.sample.js

```diff
@@ -0,0 +1,59 @@
+var DIGDAG_CONFIG = {
+  url: 'http://localhost:65432/api/',
```

Comment:
Yeah makes sense, but I don't think that we should be adding a sample config to the root where the source for the app is, i guess it doesn't matter